com.github.wwmm.easyeffects.yml

```diff
@@ -0,0 +1,389 @@
+                # note to future Flatpakers: if it's needed, webrtc-audio-processing has a module in the pulseeffects manifest that could replace this one.
+                # https://github.com/flathub/com.github.wwmm.pulseeffects/issues/44#issuecomment-885999963
+          
+          # Ladspa & LV2 Plugins
+          
+          # manifest module from: https://github.com/flathub/io.mpv.Mpv/blob/73681476e1abde88770804c497f03c4bfea027f6/io.mpv.Mpv.yml#L223
+          - name: rubberband
+            buildsystem: meson
+            cleanup:
+              - /include
+              - /lib/*.a
+              - /lib/pkgconfig
+            sources:
+              - type: archive
+                url: https://breakfastquay.com/files/releases/rubberband-1.9.2.tar.bz2
+                sha256: b3cff5968517141fcf9e1ef6b5a1fdb06a5511f148000609216cf182ff4ab612
+                x-checker-data:
+                  type: html
+                  url: https://www.breakfastquay.com/rubberband/
+                  version-pattern: Rubber Band Library v(\d\.\d+\.?\d*) source
+                  url-template: https://breakfastquay.com/files/releases/rubberband-$version.tar.bz2
+
+          - name: calf
+            build-options:
+              env:
+                - NOCONFIGURE=1
+            rm-configure: true
+            sources:
+              - type: archive
+                url: 'https://calf-studio-gear.org/files/calf-0.90.3.tar.gz'
+                sha256: e32bc23fb68f88729bdef361614fd9ac38627c774c35cd3c4eb611fca887a49f
+            post-install:
+              - install -Dm644 -t $FLATPAK_DEST/share/licenses/calf COPYING
+            cleanup:
+              - /bin
+              - /share/applications
+              - /share/bash-completion
+              - /share/calf
+              - /share/doc
+              - /share/icons
+              - /share/man
+            modules:
+              - "shared-modules/linux-audio/fluidsynth2.json"
+          
+          - name: lsp-plugins
+            no-autogen: true
+            make-args:
+              - "PREFIX=${FLATPAK_DEST}"
+              - BUILD_MODULES=lv2 # Don't build docs to avoid php dependency
+            make-install-args:
+              - "PREFIX=${FLATPAK_DEST}"
+            install-rule: install_lv2
+            build-options:
+              arch:
+                aarch64:
+                  make-args:
+                    - BUILD_PROFILE=aarch64
+                arm:
+                  make-args:
+                    - BUILD_PROFILE=armv7a
+            sources:
+              - type: git
+                url: 'https://github.com/sadko4u/lsp-plugins.git'
+                tag: 1.1.30
+                commit: d78befc2faaa6de64050c3c4b662dea429e47c40
+                x-checker-data:
+                  type: git
+                  tag-pattern: "^([\\d.]+)"
+            post-install:
+              - install -Dm644 -t $FLATPAK_DEST/share/licenses/lsp-plugins LICENSE.txt
+
+          - name: zam-plugins
```

Comment:
These are available as flatpak. Can't they just be automatically installed? (I know you can, but the question is whether it works for easyeffects)

Reply:
Not sure, I'll try it out